src/index.jsx

```diff
@@ -35,6 +37,7 @@ const createStoreWithMiddleware = applyMiddleware(
 )(createStore);
 
 const store = createStoreWithMiddleware(reducer, {
+  app: Immutable.fromJS(appReducerDefaultState),
```

Comment:
could this be empty object, and then the views could set the inital properties to the store themselves independently? is that possible? maybe via abusing `getDefaultProps` (*"Invoked once and cached when the class is created."*) ?